src/components/RadioButtonList.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import styled from "styled-components";
+import RadioListItem from "./RadioListItem"
+
+const FlexBox = styled.div`
+    display: flex;
+    flex-direction: column;
+`;
+
+const RadioButtonList = ({ options, onSelection, ...props }) => {
+    
+    // Generate identifier to group radios
+    const name = Date.now();
+
+    return (
+        <FlexBox>
+        {
+            options.map(option =>
+                <RadioListItem name={name} option={option} onSelection={onSelection} {...props}/>    
```

Comment:
React will want you to put a `key` prop on this. https://reactjs.org/docs/lists-and-keys.html

the `name` will do fine I think.